src/swap/SwapScreen.tsx

```diff
@@ -778,6 +892,77 @@ export function SwapScreen({ route }: Props) {
             appFee={appFee}
             appFeeInfoBottomSheetRef={appFeeInfoBottomSheetRef}
           />
+          {showCrossChainFeeWarning && (
+            <InLineNotification
+              variant={NotificationVariant.Warning}
+              title={t('swapScreen.crossChainFeeWarning.title', {
+                tokenSymbol: crossChainFeeInfo?.crossChainFeeToken.symbol,
+              })}
+              description={t('swapScreen.crossChainFeeWarning.body', {
+                tokenSymbol: crossChainFeeInfo?.crossChainFeeToken.symbol,
+                tokenAmount: crossChainFeeInfo?.maxCrossChainFeeMissingAmountInDecimal.toFixed(4),
```

Comment:
So it respects the user decimal separator, and default number of decimals shown. Or do we actually need 4?
```suggestion
                tokenAmount: crossChainFeeInfo?.maxCrossChainFeeMissingAmountInDecimal.toFormat(),
```

Reply:
Sure, good call!